docs/architecture.md

```diff
@@ -51,11 +51,11 @@ The above control plane components are explained in detail below.
 
 ![OpenEBS volume pods provisioning-overview](/docs/assets/volume-provisioning.png)
 
-This component runs as a POD and is core to the provisioning decisions. 
+This component runs as a POD and makes provisioning decisions. 
 
-Developer constructs a claim with the required volume parameters, chooses the appropriate storage class and invokes kubelet on the yaml specification. The OpenEBS PV dynamic provisioner interacts with the maya-apiserver to create deployment specifications for the volume controller pod and volume replica pod(s) on appropriate nodes. Scheduling of the volume pods (controller/replica) can be controlled using annotations in PVC specification, details of which are discussed in a separate section.
+The way it is used is that the developer constructs a claim with the required volume parameters, chooses the appropriate storage class and invokes kubelet on the yaml specification. The OpenEBS PV dynamic provisioner interacts with the maya-apiserver to create deployment specifications for the volume controller pod and volume replica pod(s) on appropriate nodes. Scheduling of the volume pods (controller/replica) can be controlled using annotations in PVC specification, details of which are discussed in a separate section.
```

Comment:
"invokes kubelet on the yaml specification" -> "invokes kubectl on the yaml specification"